### PR TITLE
AC-720 Redundant local variables

### PR DESCRIPTION
**Pull request by Abdelaty**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-720

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
Do follow the PR tips and dont create any PR whose ticket is not yet ready for work as it will only waste your time if mentor considers it unnecessary

**Comment by Abdelaty** [comment]
Thank you @rishabh-997  for your tip. I will take care next time.

**Comment by f4ww4z** [comment]
@Abdelaty You still did not ping me in the issue page, and also see the merge conflict.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java [hunk 1]
```diff
@@ -4,11 +4,9 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
-
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java]: This is change is not necessary.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/providerrelationship/ProviderRelationshipFragment.java [hunk 2]
```diff
@@ -4,11 +4,9 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
-
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/providerrelationship/ProviderRelationshipFragment.java]: This is change is not necessary. Imports are read better with the empty lines.

**Comment by Abdelaty** [comment]
> @Abdelaty You still did not ping me in the issue page, and also see the merge conflict.

Sorry my pc was broken last two days. And it fixed now! 
Thank you for reviewing, I will fix the conflicts now.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java (hunk 1 above)
Comment (Abdelaty) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java]: those empty lines were deleted after rearranging the code using an android studio rearrange tool. Do you want me to add them again?

File: .idea/dbnavigator.xml [hunk 3]
```diff
@@ -0,0 +1,456 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (f4ww4z) [inline .idea/dbnavigator.xml]: Ignore this file as suggested at https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore#L21

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java (hunk 1 above)
Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/providerdashboard/patientrelationship/PatientRelationshipFragment.java]: Ok, no need to change this.

File: .idea/dbnavigator.xml (hunk 3 above)
Comment (Abdelaty) [reply .idea/dbnavigator.xml]: Done I ignored it, But I have a problem with squash commits.

**Pull request merged**